### PR TITLE
established database connection

### PR DESCRIPTION
**Pull request by FelixLetheren**

### CODE REVIEW

**Comment by AlisonYoon** [comment]
looks good

File: php-files/index.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+$db = new PDO('mysql:host=192.168.20.20; dbname=fish-finder', 'root', '');
+
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$sql = $db->prepare('SELECT * FROM `fish`;');
```
Comment (sjr-mayden) [inline php-files/index.php]: SELECTing * is bad practice as it makes the query very slow, it's better to be specific as to which fields you want to SELECT

File: php-files/index.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+$db = new PDO('mysql:host=192.168.20.20; dbname=fish-finder', 'root', '');
+
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```
Comment (sjr-mayden) [inline php-files/index.php]: Extra spacing about the object accessor

File: php-files/index.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+$sql = $db->prepare('SELECT * FROM `fish`;');
+
+$sql ->execute();
```
Comment (sjr-mayden) [inline php-files/index.php]: Extra spacing about the object accessor

File: php-files/index.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+$collection = $sql -> fetchAll();
```
Comment (sjr-mayden) [inline php-files/index.php]: Extra spacing about the object accessor

File: php-files/index.php [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<?php
```
Comment (sjr-mayden) [inline php-files/index.php]: Why isn't in the root of the project, it should be

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

**Review by sjr-mayden — APPROVED** [review]

**Pull request merged**